bench/workgen/workgen.cxx

```diff
@@ -1954,6 +2050,9 @@ WorkloadOptions::WorkloadOptions() : max_latency(0),
       "When set to the empty string, no JSON is emitted.");
     _options.add_int("sample_interval_ms", sample_interval_ms,
       "performance logging every interval milliseconds, 0 to disable");
+    _options.add_int("max_idle_table_cycle", max_idle_table_cycle,
+      "Value is the maximum number of seconds a create or drop is allowed"
```

Comment:
To match the other entries, I would start this `"maximum number of seconds...".`  Also you're missing a space, this output will read `"...is allowedbefore aborting..."`